indico/modules/events/registration/client/js/form_setup/ItemSettingsModal.jsx

```diff
@@ -145,6 +145,17 @@ export default function ItemSettingsModal({id, sectionId, defaultNewItemType, on
             />
           )}
           {SettingsComponent && <SettingsComponent {...itemData} />}
+          <FinalInput
+            name="retentionPeriod"
+            type="number"
+            placeholder={Translate.string('Permanent')}
+            step="1"
+            min="0"
+            validate={v.optional(v.min(0))}
+            parse={val => (val === '' ? 0 : val)}
+            format={val => (val === 0 ? '' : val)}
+            label={Translate.string('Retention period (months)')}
```

Comment:
I remember we talked about this at some point...,hadn't we decided on a *weeks* granularity for this?